tools/patch/grammar.patch

```diff
@@ -0,0 +1,833 @@
+diff --git a/Grammar/python.gram b/Grammar/python.gram
```

Comment:
I wonder how maintainable this is.

Maybe we should write a script that changes the grammar instead of maintaining the patch I don't know

Reply:
which do you think is more maintainable?

Reply:
Lets see what happens when we go to 3.10.X or 3.9.>5